bookkeeper-common/src/main/java/org/apache/bookkeeper/common/conf/ComponentConfiguration.java

```diff
@@ -287,4 +291,31 @@ public String getString(String key, String defaultValue) {
     public List<Object> getList(String key, List<?> defaultValue) {
         return conf.getList(getKeyName(key), defaultValue);
     }
+
+    /**
+     * returns the string representation of json format of this config.
+     *
+     * @return
+     * @throws ParseJsonException
+     */
+    public String asJson() {
```

Comment:
but where is this used?

Reply:
it is used in a subsequent pull request: https://github.com/apache/bookkeeper/pull/1553/files#diff-3f4d443c559ae8b071b0229fbb91df2aR209

I am breaking the changes into small changes, so they are easier to be reviewed.